webcurator-ui/src/components/LoginDialog.vue

```diff
@@ -0,0 +1,83 @@
+
+<template>
+	<div>
+        <!-- <div class="text-center mb-5" style="padding: 0px 100px 5px 100px;">
+            <img src="@/assets/wct_logo.png" alt="Image" height="50" class="mb-3" />            
+        </div> -->
+
+        <div class="flex align-items-center gap-3 mb-3">
+            <label for="username" class="font-semibold w-6rem">Username</label>
+            <br/>
+            <InputText id="username" type="text" placeholder="Username" class="w-full mb-3" v-model="username"/>
+        </div>
+        <br/>
+        <div class="flex align-items-center gap-3 mb-3">
+            <label for="password" class="font-semibold w-6rem">Password</label>
+            <br/>
+            <InputText id="password" type="password" placeholder="Password" class="w-full mb-3" v-model="password"/>
+        </div>
+
+        <br/>
+        <Divider />
+        <br/>
+
+        <div class="flex justify-content-end gap-2">
+            <Button type="button" icon="pi pi-user" label="login"  class="w-full p-3 text-xl" @click="auth"></Button>
+        </div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted, inject } from "vue";
+// import { useToast } from 'primevue/usetoast';
+// const toast = useToast();
+
+const dialogRef:any = inject("dialogRef");
+const username=ref(null);
+const password=ref(null);
+
+
+const auth = () => {
+    var url="./auth/v1/token?username=" + username.value + "&password=" + password.value;
```

Comment:
@leefrank9527 is this putting a password in the query string that could be seen in the HTTP headers?

Reply:
> @leefrank9527 is this putting a password in the query string that could be seen in the HTTP headers?

Yes. The credential is included in the URL, I would put it inside the post body instead.

Reply:
Ok. And its already encrypted right?

Reply:
> Ok. And its already encrypted right?

It dependends on the transfer protocol. If the https is used, the credential in the http body will be encryped. If the http is used, the credential is still plain text. 

Have updated the branch to put the credential in the http body.